website/docs/r/cloud_l4_l7_native_load_balancer.html.markdown

```diff
@@ -0,0 +1,118 @@
+  is_copy                       = "no"
+  is_instantiation              = "no"
+  is_static_ip                  = "no"
+  managed                       = "no"
+  prom_mode                     = "no"
+  scheme                        = "internal"
+  size                          = "medium"
+  sku                           = "standard"
+  service_type                  = "NATIVELB"
+  target_mode                   = "primary"
+  trunking                      = "no"
+  cloud_l4l7_load_balancer_type = "application"
+  instance_count                = "2"
+  max_instance_count            = "10"
+  min_instance_count            = "5"
+}
+```
+
+## Argument Reference ##
+
+* `tenant_dn` - (Required) Distinguished name of the parent Tenant object. Type: String.
+* `name` - (Required) Name of the Cloud L4-L7 Native Load Balancer object. Type: String.
+* `annotation` - (Optional) Annotation of the Cloud L4-L7 Native Load Balancer object. Type: String.
+* `name_alias` - (Optional) Name Alias of the Cloud L4-L7 Native Load Balancer object. Type: String.
+* `version` - (Optional) Version of the Cloud L4-L7 Native Load Balancer object. Default value is "1.0". Type: String.
+* `active_active` - (Optional) Active-Active mode of the Cloud L4-L7 Native Load Balancer object. Allowed values are "no", "yes", and default value is "no". Type: String.
+* `allow_all` - (Optional) Allow-All of the Cloud L4-L7 Native Load Balancer object. Allowed values are "no", "yes", and default value is "no". Type: String.
+* `auto_scaling` - (Optional) Auto-Scaling of the Cloud L4-L7 Native Load Balancer object. Allowed values are "no", "yes", and default value is "no". Type: String.
+* `context_aware` - (Optional) A value to determine if the L4-L7 device cluster supports multiple contexts (VRFs). Allowed values are "multi-Context", "single-Context", and default value is "single-Context". Type: String.
+* `custom_rg` - (Optional) Custom RG of the Cloud L4-L7 Native Load Balancer object. Type: String.
+* `device_type` - (Optional) Device Type of the Cloud L4-L7 Native Load Balancer object. Allowed values are "CLOUD", "PHYSICAL", "VIRTUAL", and default value is "CLOUD". Type: String.
+* `function_type` - (Optional) Function Type of the Cloud L4-L7 Native Load Balancer object. Allowed values are "GoThrough", "GoTo", "L1", "L2", "None", and default value is "GoTo". Type: String.
+* `instance_count` - (Optional) Instance Count of the Cloud L4-L7 Native Load Balancer object. Default value is "2". Type: String.
+* `is_copy` - (Optional) Is the device is a copy device. Allowed values are "no", "yes", and default value is "no". Type: String.
+* `is_instantiation` - (Optional) Is Instantiation of the Cloud L4-L7 Native Load Balancer object. Allowed values are "no", "yes", and default value is "no". Type: String.
+* `is_static_ip` - (Optional) Is Static IP of the Cloud L4-L7 Native Load Balancer object. Allowed values are "no", "yes", and default value is "no". Type: String.
```

Comment:
rephrase. this sounds like a question 

ex: Enables instatntiation of the cloud...